docs/docusaurus/docs/cloud/connect/connect_snowflake.md

```diff
@@ -0,0 +1,208 @@
+
+To learn how to deploy a Docker container image in a specific environment, see the following documentation:
+
+- [Quickstart: Deploy a container instance in Azure using the Azure CLI](https://learn.microsoft.com/en-us/azure/container-instances/container-instances-quickstart)
+
+- [Build and push a Docker image with Google Cloud Build](https://cloud.google.com/build/docs/build-push-docker-image)
+
+- [Deploy Docker Containers on Amazon ECS](https://aws.amazon.com/getting-started/hands-on/deploy-docker-containers/)
+
+You can deploy the GX Agent in any environment in which you create Kubernetes clusters. For example:
+
+- [Amazon Elastic Kubernetes Service (EKS)](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html)
+
+- [Microsoft Azure Kubernetes Service (AKS)](https://learn.microsoft.com/en-us/azure/architecture/reference-architectures/containers/aks-start-here)
+
+- [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine/docs)
+
+- Any Kubernetes cluster version 1.21 or greater which uses standard Kubernetes
+
+<Tabs
+  groupId="connect-snowflake"
+  defaultValue='docker'
+  values={[
+  {label: 'Docker', value:'docker'},
+  {label: 'Kubernetes', value:'kubernetes'},
+  ]}>
+<TabItem value="docker">
+
+1. Download the GX Agent Docker container image from [Docker Hub](https://hub.docker.com/r/greatexpectations/agent).
+
+2. After configuring your cloud service to run Docker containers, run the following Docker command to start the GX Agent: 
+
+   ```bash title="Terminal input"
+   docker run -it \
+   -e GX_CLOUD_ACCESS_TOKEN= YOUR_ACCESS_TOKEN \ 
+   -e GX_CLOUD_ORGANIZATION_ID= YOUR_ORGANIZATION_ID \  
+   greatexpectations/agent:latest
+    ```
+    Replace `YOUR_ACCESS_TOKEN` and `YOUR_ORGANIZATION_ID` with the values you copied previously. To store your Snowflake password as an environment variable, add `-e GX_CLOUD_SNOWFLAKE_PW= YOUR_SNOWFLAKE_PASSWORD \` to the Docker command.
+
+3. Run the following command to use the GX Agent image as the base image and optionally add custom commands:
+
+   ```bash title="Terminal input"
+   FROM greatexpectations/agent
+   RUN echo "custom_commands"
+   ```
+4. Run the following command to rebuild the Docker image:
+
+   ```bash title="Terminal input"
+   docker build -t myorg/agent
+   ```
+
+</TabItem>
+<TabItem value="kubernetes">
+
+1. Install kubectl. See [Install Tools](https://kubernetes.io/docs/tasks/tools/).
+
+2. Run the following command to provide the access credentials to the Kubernetes container:
+    
+   ```sh
+   kubectl create secret generic gx-agent-secret \
+   --from-literal=GX_CLOUD_ORGANIZATION_ID=YOUR_ORGANIZATION_ID \
+   --from-literal=GX_CLOUD_ACCESS_TOKEN=YOUR_ACCESS_TOKEN \
+   ```
+    Replace `YOUR_ORGANIZATION_ID` and `YOUR_ACCESS_TOKEN` with the values you copied previously. To include your Snowflake password as an access credential, add `--from-literal=GX_CLOUD_SNOWFLAKE_PW=YOUR_SNOWFLAKE_PASSWORD` to the command.
```

Comment:
If we are no longer passing in PW as part of the ENV, then the second sentence about `GX_CLOUD_SNOWFLAKE_PW` will be going away